_env/nginx.conf

```
@@ -43,7 +43,8 @@ http {
     }
 
     location /forum/ {
-      proxy_pass https://patience.studentrobotics.org/forum/;
+      # proxy_pass https://patience.studentrobotics.org/forum/;
+      return 307 https://studentrobotics.org/docs/team_admin/discord;
```

Suggest doing this instead, to avoid any hint of recursion (and also to allow the underlying target to more easily be changed):
```suggestion
      return 307 /docs/team_admin/discord;
```